F/FFLAS_FFPACK/build_tarballs.jl

```diff
@@ -30,10 +30,11 @@ install_license COPYING
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = supported_platforms(exclude=Sys.iswindows)
+platforms = [AnyPlatform()]
```

Comment:
I don't think this is correct, apparently there were some diffences between 32-bit and 64-bit platforms